control-plane/README.md

```diff
@@ -71,7 +71,7 @@ The port `8443` can be adjusted as need for your local system.
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.71.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.0.0 |
```

Comment:
The `rke1` module has an `aws_provider` setting does it get impacted by this update?

Reply:
Yes, good call out. I'm working on making the necessary changes for another PR to account for RKE1 and K3S module updates for the aws 4.x provider